scripts/pkg/build_templates/current/opensearch-dashboards/rpm/opensearch-dashboards.rpm.spec

```diff
@@ -123,7 +123,6 @@ exit 0
 %license %{product_dir}/LICENSE.txt
 %{product_dir}/package.json
 %{product_dir}/manifest.yml
-%{product_dir}/.i18nrc.json
```

Comment:
Is this change compatible with 1.x line as well?

Reply:
1.x line templates are in `legacy` folder so not affected at all.
This change will only take place in 2.18+.